test/api/v3/unit/models/task.test.js

```diff
@@ -151,6 +151,169 @@ describe('Task Model', () => {
       });
     });
 
+    describe('sanitizeTasks', () => {
```

Comment:
what's the reasons for this new tests? they seem to validate that valid data does not error so it doesn't say anything about whether validation works (and I think we already have tests for those behaviors)... Can you instead add tests for this new validation?

Reply:
@paglias Sorry, I don't think I understand correctly. Many of these tests didn't exist before so I added them to increase coverage. For todo/reward/habit they only check valid inputs, only because they aren't directly related to this ticket. For dailies I have tests against negative ints, non ints, and valid inputs. One I do see I'm missing is large ints which I can add. But what kind of test are you thinking of?

Reply:
> For dailies which is what this issue relates to I have tests against negative ints, non ints, and valid inputs. One I do see I'm missing is large ints which I can add. But what kind of test are you thinking of?

Sorry I didn't notice you already added them , they're enough but could you move them to the integration folder since most validation tests are there (under `tasks/POST_tasks_user.test.js`)?

>  Many of these tests didn't exist before so I added them to increase coverage. For todo/reward/habit they only check valid inputs, only because they aren't directly related to this ticket,

What I don't think is quite right is that in these new tests ("sanitize x") you're checking that validation doesn't fail if the data is correct which is not what we should test for two reasons IMHO:

- there should be a clear thing that is tested like "fails if input is X" not not fail if input is different from X 
- these tests are difficult to maintain as we should pass new data to the task creation every time a new field is added to the task model

Reply:
Alright sure, I'll update these. Thanks!